sample_data/example-annotation.ttl

```diff
@@ -0,0 +1,123 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix ns1: <http://www.w3.org/ns/oa#> .
+@prefix ns2: <http://mu.semte.ch/vocabularies/ext/> .
+@prefix ns3: <https://data.vlaanderen.be/ns/slimmeraadpleegomgeving#> .
+@prefix ns4: <http://www.w3.org/ns/prov/> .
+@prefix ns5: <https://id.erfgoed.net/vocab/ontology#> .
+@prefix ns6: <http://purl.org/ontology/bibo/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix void: <http://rdfs.org/ns/void#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://mu.semte.ch/vocabularies/ext/maatregeltypes/measureTypes> a skos:ConceptScheme ;
+    skos:hasTopConcept <http://mu.semte.ch/vocabularies/ext/maatregeltypes/1>,
+        <http://mu.semte.ch/vocabularies/ext/maatregeltypes/2>,
+        <http://mu.semte.ch/vocabularies/ext/maatregeltypes/3> ;
+    skos:prefLabel "Measure types" .
+
+<https://id.erfgoed.net/besluiten/14767> a <http://data.vlaanderen.be/ns/besluit#Besluit>,
+        ns5:Besluit ;
+    dcterms:hasPart <https://id.erfgoed.net/besluiten/14767/bestanden/23877>,
+        <https://id.erfgoed.net/besluiten/14767/bestanden/23878>,
+        <https://id.erfgoed.net/besluiten/14767/bestanden/23879>,
+        <https://id.erfgoed.net/besluiten/14767/bestanden/23880> ;
+    dcterms:identifier 14767 ;
+    dcterms:publisher <http://example.org/.well-known/genid/2> ;
+    dcterms:source <http://www.ejustice.just.fgov.be/cgi/summary_body.pl?language=nl&pub_date=2019-03-20> ;
+    dcterms:spatial <http://example.org/.well-known/genid/4> ;
+    dcterms:title "Architectenwoning Louis Hagen" ;
+    void:inDataset <https://id.erfgoed.net/datasets/besluiten> ;
+    skos:changeNote <http://example.org/.well-known/genid/10>,
+        <http://example.org/.well-known/genid/6>,
+        <http://example.org/.well-known/genid/8> ;
+    skos:editorialNote ""@nl ;
+    ns5:besluit_handeltOver <https://inventaris.onroerenderfgoed.be/aanduidingsobjecten/113477> ;
+    ns5:besluit_uitgevaardigdAls <https://id.erfgoed.net/thesauri/besluittypes/9> ;
+    ns5:dateBetekend "2019-02-21"^^xsd:date ;
+    ns5:dateSigned "2019-02-21"^^xsd:date ;
+    ns5:precedes <https://id.erfgoed.net/besluiten/14850> ;
+    ns5:rechtsgevolgen <https://id.erfgoed.net/thesauri/besluittypes/12> .
+
+<https://id.erfgoed.net/besluiten/14767/bestanden/23878/annotation/890> a ns1:Annotation ;
+    ns1:hasTarget <https://id.erfgoed.net/besluiten/14767/bestanden/23878/measure/123> .
+
+<https://id.erfgoed.net/besluiten/14767/bestanden/23879/annotationActivity> a ns4:Activity,
+        <https://www.w3.org/ns/activitystreams#Create> ;
+    ns4:wasAssociatedWith <http://example/com/actor/hackathon-oe-ai> ;
+    ns3:Activiteit.genereertAnnotatie <https://id.erfgoed.net/besluiten/14767/bestanden/23879/annotation/890> .
```

Comment:
RDFlib doesn't like this predicate. I "_" it

Reply:
I took this into account in the script. Ok to just replace it for a hackathon :)